### ISSUE TITLE
Confusion in Age Transformation Loss

### ISSUE THREAD

**Issue by HowToNameMe** [yoyohonyang/LearingFaceAgeProgression#1]
Can you show us your source code in training phase? I think the highlight of the paper lies in its age transformation loss which is confusing according to your description in paper.

**Comment by zxx001** [comment]
I am confused by how to achieve identity preservation loss while training ,could you please send me this part code or show us the training code?thank you .

**Comment by wang21jun** [comment]
anyone  deal with this work? can we discuss something?
